.travis.yml

```diff
@@ -1,5 +1,5 @@
 dist: trusty
-sudo: false
+sudo: required
```

Comment:
> It can invoke docker but the builds themselves can no longer run inside a docker container

We dont run docker ourselves, I meant the second option.

> One question is how much slower if at all are the tests going to be if they run in Travis's normal systems as opposed inside their docker containers?

Depends, could be a couple of seconds per option in the matrix
https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments
https://docs.travis-ci.com/user/migrating-from-legacy/#Builds-start-in-seconds